web-api/src/storage.rs

```diff
@@ -188,14 +190,23 @@ impl Config {
     pub(crate) async fn setup(&self) -> Result<Storage, SetupError> {
         let elastic = self.elastic.setup_client()?;
         let postgres = self.postgres.setup_database().await?;
-
-        Ok(Storage { elastic, postgres })
+        #[cfg(feature = "ET-3837")]
+        let is_migrated = RwLock::new(false);
```

Comment:
AFIK this seem to be currently not doing anything as it's only set by one specif binary before exiting.

Reply:
okay I need to check my network time sync settings, it just showed that your message _will_ be send in 14 seconds, with a nice count down to now :rofl: